services/storage-users/pkg/command/uploads.go

```diff
@@ -223,6 +250,16 @@ func ListUploadSessions(cfg *config.Config) *cli.Command {
 						u.Expires().Format(time.RFC3339),
 						strconv.FormatBool(u.IsProcessing()),
 					})
+
+					if c.Bool("restart") {
+						if err := events.Publish(context.Background(), stream, events.ResumePostprocessing{
+							UploadID:  u.ID(),
+							Timestamp: utils.TSNow(),
+						}); err != nil {
+							fmt.Fprintf(os.Stderr, "Failed to send restart event for upload session '%s'\n", u.ID())
```

Comment:
can we add a `--continue` or `--fail-on-error` parameter to allow continuing with the next upload?

Reply:
Does that make sense? The only thing that could happen here is that nats is not available. Hence other uploads will also fail. The command is too fast for nats to recover between two uploads.

Reply:
GNU recommends `--ignore-errors` and the shorthand `-i`: https://www.gnu.org/prep/standards/html_node/Option-Table.html#Option-Table

Reply:
Does it really need to be that complicated? Can we just pick one way and go with it?

Reply:
I just don't want to be bitten by this. We can get this in without the `--ignore-errors` ... but future me is worried, even if the error now only indicates a connection to nats is not possible.

Reply:
Present me likes simplicity :smile: Future me will take care if your prophecy comes true